### PR TITLE
[Tests] WebExtension loading Validation

### PR DESCRIPTION
**Pull request by Joacchim**
As defined in #36, this PR:
 - Rework the baseline test and utilities to factorize the Driver setup
 - Add a test to load each supported URL to validate that the extension loads properly
 - Improve testing setup to provide additional feedback in case of failure (output, dump of the browser's console logs in case of failure)
 - Add a test to load an unsupported URL and validate that the extension is not loaded for unplanned urls.

Please review commits starting at f3cecbf (This is to be rebased over and merged after #32)

### CODE REVIEW

**Comment by Joacchim** [comment]
To be merged after #32

File: tests/func/utils/drivers/chrome.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+from .base import BaseWebdriverWrapper
+
+class Wrapper(BaseWebdriverWrapper):
+
+    def __init__(self, *args, **kwargs):
+        super(Wrapper, self).__init__(*args, **kwargs)
+
+        capabilities = DesiredCapabilities.CHROME
+        capabilities['loggingPrefs'] = { 'browser':'ALL' }
```
Comment (GuillaumeGomez) [inline tests/func/utils/drivers/chrome.py]: nit: `{'browser': 'ALL'}`

File: tests/conftest.py [hunk 2]
```diff
@@ -18,6 +16,15 @@ def pytest_generate_tests(metafunc):
         metafunc.parametrize('webdriver', webdrivers)
 
 
+def pytest_exception_interact(node, call, report):
+    # import pdb; pdb.set_trace()
+    if report.failed:
+        # Retrieve test's browser logs through the webdriver
+        print('')
```
Comment (GuillaumeGomez) [inline tests/conftest.py]: I'm thinking about something like this instead:

```python
# Retrieve test's browser logs through the webdriver
s = '\n'.join([line for line in node.funcargs['webdriver'].get_log('browser')])
if len(s) > 0:
    print('=== LOGS ===')
    print(s)
```

File: tests/func/utils/drivers/chrome.py (hunk 1 above)
Comment (Joacchim) [reply tests/func/utils/drivers/chrome.py]: fixed.

File: tests/conftest.py (hunk 2 above)
Comment (Joacchim) [reply tests/conftest.py]: Included :)

**Pull request merged**